examples/lidar_obstacles/include/lds01_params.h

```diff
@@ -33,7 +33,7 @@ static const lds01_params_t lds01_params[] = {
     {
         .uart = UART_DEV(2),
         .new_frame_cb = new_frame_available_cb,
-        .invert_data = true
+        .invert_data = false
```

Comment:
same here, do not change behavior in c++ syntax refactoring commit

Reply:
It's a mistake, I don't understand why it has changed...